katalyze/main.go

```diff
@@ -41,6 +41,8 @@ import (
 
 var (
 	outputDir       = flag.String("output_dir", "", "Directory for returning the processed SGFs. By default, uses current directory")
+	bucket          = flag.String("bucket", "", "The bucket for CloudStorage of analyzed games. This param is optional.")
+	bucketPrefix    = flag.String("bucket_prefix", "", "Dictates the path within the bucket where games wil be stored. Required if the bucket parameter is supplied.")
```

Comment:
Not sure I understand bucket prefix?

Reply:
At least for AWS, you usually specify the bucket and the path within the bucket separately. Might be different for GCP

Reply:
Yeah, that's true for GCS also. maybe root folder?

Reply:
Or maybe prefix is canonical

Reply:
I think prefix is canonical